eng/common/pipelines/templates/steps/git-push-changes.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  PRBranchName: not-specified
+  PROwner: azure-sdk
+  CommitMsg: not-specified
+  RepoOwner: Azure
+  RepoName: $(Build.Repository.Name)
+  PushArgs:
+  WorkingDirectory: $(System.DefaultWorkingDirectory)'
+  ScriptDirectory: eng/common/scripts
+  SkipCheckingForChanges: false
+  RemoteName: not-specified
```

Comment:
This should default to `azure-sdk-fork` (the default in `git-branch-push.ps1`) so calls to this template from `create-pull-request.yml` (which doesn't have a `RemoteName` parameter) don't change behavior when this checks in.

Reply:
If it calls create-pull-request, then `RemoteName` here did not get set. The value will default to `azure-sdk-fork` to `git-branch-push.ps1`, right? This is the same behavior as before.

Reply:
Currently you would be passing not-specified as the value which is not correct. You should either leave it empty or default it to a valid value as Daniel suggests.

Reply:
Sure. I am having a test, and make sure the default value is correct if it is not setting.